docs/adding-a-page.md

```diff
@@ -0,0 +1,325 @@
+# addons-frontend
+
+This will outline what is required to add a page to the project. A basic knowledge of
+[react](https://facebook.github.io/react/docs/getting-started.html) and
+[redux](http://redux.js.org/) is assumed.
+
+## Structure
+
+A basic app structure will look like this:
+
+```
+src/
+  <app-name>/
+    components/
+      MyComponent/
+        index.js
+        styles.scss
+    containers/
+      MyContainer/
+        index.js
+        styles.scss
```

Comment:
I've been putting my CSS next to the JS it belongs to. We should pick one way though. Maybe we should make a list of things we haven't been consistent on and go through it and enforce stuff with eslint/tests.